proto/viam/app/v1/app.proto

```diff
@@ -870,11 +883,14 @@ message ListFragmentsResponse {
 message GetFragmentRequest {
   string id = 1;
   string current_organization_id = 2;
+  optional string version = 3;
 }
 
 message GetFragmentResponse {
   Fragment fragment = 1;
-  FragmentUsage fragment_usage = 2;
+  repeated FragmentUsage usages = 2;
+  repeated FragmentVersion versions = 3;
+  repeated FragmentTag tags = 4;
```

Comment:
Should these be nested in the Fragment message? Or are you doing this to only return these fields on GetFragment?

Reply:
good q. looking at other uses of `Fragment`, if i nest it in `Fragment` the tags will also be returned with:
`GetFragmentHistory`
`ListFragments`
`CreateFragment`
`UpdateFragment`

do we think we'll want tag info for all of these endpoints? happy to nest if so

Reply:
Yeah I think for consistency lets put into the Fragment message. The size of the data transmitted is not going to increase that much. We might not use the fields on app.viam.com in some APIs but I think it will be useful for any API customers and prevent having to call Get after List, for example.

Reply:
moved tags into fragment!